src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
so just so I understand your perspectives correctly: 
It sounds like it's understood that Akira's suggestion of using `.bx--breadcrumb-item:last-child::after` works because it will automatically remove the trailing slash off any last breadcrumb item. 

But what @rahulrkr08 is saying that a dev can add the proposed utility class `.bx--breadcrumb-item--last` so a dev can either opt in or opt out of having a trailing slash, correct?